.azure/pipelines/ci.yml

```diff
@@ -780,12 +762,62 @@ stages:
             - Linux_musl_x64_build
             - Linux_musl_arm_build
             - Linux_musl_arm64_build
-          # In addition to the dependencies above, ensure the build was successful overall.
+          # In addition to the dependencies above that provide assets, ensure the build was successful overall.
+          - ${{ if in(variables['Build.Reason'], 'Manual') }}:
+            - Code_check
+            - Windows_Test
+            - MacOS_Test
+            - Linux_Test
+            - Helix_x64
           - Source_Build_Managed
         pool:
           vmImage: vs2017-win2016
         publishUsingPipelines: ${{ variables._PublishUsingPipelines }}
         enablePublishBuildArtifacts: true # publish artifacts/log files
+  - ${{ and(ne(variables['System.TeamProject'], 'public'), eq(variables['Build.SourceBranch'],'refs/heads/main')) }}:
```

Comment:
Test build is https://dev.azure.com/dnceng/internal/_build/results?buildId=1388396&view=results. That lacks this line, meaning my manual build should do the indexing (again). In the future, only official builds or manual builds of the 'main' branch will do this.

Nice side effect of this line is we won't have to remove anything when branching from main in the future.